internal/controllers/common/resource_definitions/resource_definitions.go

```diff
@@ -526,6 +545,24 @@ func NewPodForReports(cr *operatorv1beta1.Cryostat, imageTags *ImageTags, tls *T
 		}
 	}
 
+
+	var nodeSelector map[string]string
+	var affinity *corev1.Affinity
+	var tolerations []corev1.Toleration
+	var schedulingOptions = cr.Spec.ReportOptions.SchedulingOptions
+
+	if schedulingOptions != nil {
```

Comment:
After the rebase, it seems that we can no longer assume that `cr.Spec.ReportOptions` is non-nil here. This is why the unit tests are panicking. This snippet below fixes the problem:

```suggestion

	if cr.Spec.ReportOptions != nil && cr.Spec.ReportOptions.SchedulingOptions != nil {
		schedulingOptions := cr.Spec.ReportOptions.SchedulingOptions
```